### PR TITLE
Danielhauschildt/changelogs

### PR DESCRIPTION
**Pull request by DanielHauschildt**
Added changelog generator

### CODE REVIEW

File: packages/node/changelog/1.2.1/20230811184244-Changelog item.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+type: Added
+description: |
+  `CHANGELOG.md` for an better overview of the changes.
```
Comment (mirko314) [inline packages/node/changelog/1.2.1/20230811184244-Changelog item.yml:4]: The file name contains a space, is this expected>

File: packages/web/changelog/1.2.1/20230811184244-Changelog item.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+type: Added
+description: |
+  `CHANGELOG.md` for an better overview of the changes.
```
Comment (mirko314) [inline packages/web/changelog/1.2.1/20230811184244-Changelog item.yml:4]: Same as above

File: packages/web/package.json [hunk 3]
```diff
@@ -38,24 +38,29 @@
...
     "lint": "npx prettier --write ."
   },
   "dependencies": {
-    "onnxruntime-web": "^1.16.0",
     "@types/lodash": "^4.14.195",
     "@types/node": "^20.3.1",
     "lodash": "^4.17.21",
     "ndarray": "^1.0.19",
+    "onnxruntime-web": "^1.16.0",
+    "yaml": "^2.3.4",
```
Comment (mirko314) [inline packages/web/package.json:63]: Is yaml a real dependency or a devDependency?

File: packages/node/changelog/1.2.1/20230811184244-Changelog item.yml (hunk 1 above)
Comment (DanielHauschildt) [reply packages/node/changelog/1.2.1/20230811184244-Changelog item.yml:4]: It's allowed. Yes

File: packages/web/package.json (hunk 3 above)
Comment (DanielHauschildt) [reply packages/web/package.json:63]: Good catch

File: packages/node/changelog/1.2.1/20230811184244-Changelog item.yml (hunk 1 above)
Comment (DanielHauschildt) [reply packages/node/changelog/1.2.1/20230811184244-Changelog item.yml]: I renmaed it though

Comment (DanielHauschildt) [reply packages/node/changelog/1.2.1/20230811184244-Changelog item.yml]: I renmaed it though

**Review by mirko314 — APPROVED** [review]

**Pull request merged**